src/ui/public/toggle_panel/toggle_panel.js

```diff
@@ -12,7 +12,8 @@ app.directive('togglePanel', function () {
     template: template,
     scope: {
       togglePanelId: '@',
-      text: '@',
+      toggleButtonLabel: '=',
```

Comment:
I don't like the name of this property. It exposes too much about how `toggle-panel` is implemented (namely that it has a child component of `toggle-button`.) `text` was a better property name, IMO.